examples/restify/tracer.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/api');
+
+const { diag, DiagConsoleLogger, DiagLogLevel } = opentelemetry;
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.VERBOSE);
+
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const { SimpleSpanProcessor, ConsoleSpanExporter } = require('@opentelemetry/tracing');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+
+const { RestifyInstrumentation: Instrumentation } = require('../../plugins/node/opentelemetry-instrumentation-restify');
```

Comment:
why renaming this to `Instrumentation` ?
can you also use `@opentelemetry/instrumentation-restify` instead as well as `@opentelemetry/instrumentation-http` the last path probably exists only on yr local env.

Reply:
> why renaming this to Instrumentation?

To say "in this context, this is **the** instrumentation we're introducing to you, dear developer". It also makes less ctrl-f-replacing if the following code is moved somewhere.

> can you also use @opentelemetry/instrumentation-restify

`@opentelemetry/instrumentation-restify` simply does not exist and would be broken until a publish. #422

> @opentelemetry/instrumentation-http the last path probably exists only on yr local env.

Good catch! Fixed that.

Reply:
if you use lerna for that it will work fine pointing to yr local version instead of npm where this package is not released. We are doing like this in our all examples in core and contrib repo and it works fine.

Reply:
I don't think renaming it is a good idea, generally users will have multiple instrumentations so they will almost never does this directly

Reply:
I think this need be adressed before merging, we discussed this between maintainers and we expect users to be able to just copy/paste example together. Having to rename the import to do so add a friction that isn't needed

Reply:
> if you use lerna for that it will work fine pointing to yr local version instead of npm where this package is not released. We are doing like this in our all examples in core and contrib repo and it works fine.

Examples are not linked. So it will not work until the first publish and will only ever work with the latest published release of the instrumentation. I broke this for consistency, but imo this is not OK.